src/pint/utils.py

```diff
@@ -1667,3 +1667,17 @@ def remove_dummy_distance(c):
             "Do not know coordinate frame for %r: returning coordinates unchanged" % c
         )
         return c
+
+
+def info_string(prefix_string="# ", comment=None):
```

Comment:
How does this handle comments containing newlines?

Reply:
I think it's OK (it converts `\n` to `\\n`) but I don't know if broader sanitation is needed.

Reply:
Well, I guess my question is, if I supply a multiline comment, do I get:
```
C Comment: line one
line two
line three
```
or
``` 
C Comment: line one
C Comment: line two
C Comment: line three
```
or something else?

A test case to check (and specify for the curious) would be good.